engine/src/main.cpp

```diff
@@ -240,26 +233,7 @@ bool isVista = false;
 
 Unit *TheTopLevelUnit;
 
-/*
-// disabled as does not compile with boost 1.64
-void initLogging(char debugLevel){
-    auto loggingCore = boost::log::core::get();
-
-    switch (debugLevel) {
-    case 1:
-        loggingCore->set_filter(boost::log::trivial::severity >= boost::log::trivial::info);
-        break;
-    case 2:
-        loggingCore->set_filter(boost::log::trivial::severity >= boost::log::trivial::debug);
-        break;
-    case 3:
-        loggingCore->set_filter(boost::log::trivial::severity >= boost::log::trivial::trace);
-        break;
-    default:
-        loggingCore->set_filter(boost::log::trivial::severity >= boost::log::trivial::warning);
-        break;
-    }
-}*/
```

Comment:
Hey, this was my code! Who took it out, and when?

Reply:
The code in main was commented out and it wasn't clear if this was a recent attempt to add logs or an ancient one. I don't mind reverting it. 
Regarding VDU damages, I'll need a lot more time. The damages enum was part of the networking section, but now appears to play a part in non-networking code. I'll need to go over every reference to it to figure out what to keep. Yay.